deps/SCsub

```diff
@@ -16,5 +16,12 @@ def build_lexy_vdf(env):
     env.Append(CPPPATH=lvdf_env.lexy_vdf["INCPATH"])
     env.lexy_vdf = lvdf_env.lexy_vdf
 
+def build_ordered_map(env):
+    include_path = "ordered-map/include"
+    env.ordered_map = {}
+    env.ordered_map["INCPATH"] = [env.Dir(include_path)]
+    env.Append(CPPPATH=env.ordered_map["INCPATH"])
```

Comment:
In order for the main repo to be able to build, it needs access to the `ordered_map` headers. This can be done by adding ` + env.ordered_map["INCPATH"]` to line 66 of this repo's `SConstruct`, so it reads like so:
```
env.openvic_simulation["INCPATH"] = [env.Dir(include_path)] + env.openvic_dataloader["INCPATH"] + env.ordered_map["INCPATH"]
```

Reply:
Better just to expose that to SCsub.